src/main/webapp/assets/readfile.css

```diff
@@ -10,5 +10,9 @@
   margin: auto;
   padding: 50px;
   width: 500px;
+  background-color: #efefef
```

Comment:
change color to be RGB to match the way we reference color throughout the project.